src/Diagnostics.RuntimeHost/Controllers/ControllerBase.cs

```diff
@@ -126,11 +126,11 @@ protected OperationContext<TResource> PrepareContext<TResource>(TResource resour
             return Ok(queryRes);
         }
 
-        protected async Task<HostingEnvironment> GetHostingEnvironment(string subscriptionId, string resoourceGroup, string name, DiagnosticStampData stampPostBody, DateTime startTime, DateTime endTime)
+        protected async Task<HostingEnvironment> GetHostingEnvironment(string subscriptionId, string resourceGroup, string name, DiagnosticStampData stampPostBody, DateTime startTime, DateTime endTime)
```

Comment:
nitpik: Move to HostingEnvironmentControllerBase class

Reply:
naa. This is needed by SiteController as well.